tests/cli/test_rasa_train.py

```diff
@@ -31,7 +31,19 @@
 import rasa.utils.io
 
 
-def test_train(run_in_simple_project: Callable[..., RunResult], tmp_path: Path):
+@pytest.mark.parametrize(
+    "optional_arguments",
+    [
+        ["--endpoints", "endpoints.yml"],
+        ["--endpoints", "non_existent_endpoints.yml"],
```

Comment:
Shouldn't this raise some error if the file doesn't exist? 🤔
I think the fixture `run_in_simple_project` creates an `init` project which will contain by default `endpoints.yml`

Reply:
It doesn't raise an error because the `train` process doesn't process the file at all.  For this test, I wanted to make sure that if in the future endpoints are processed during training, we would notice immediately that this test fails because it now cares about existent/non-existent endpoints.